src/game.cpp

```diff
@@ -3725,7 +3725,8 @@ std::vector<std::string> game::list_active_characters()
  */
 void game::write_memorial_file(std::string sLastWords)
 {
-    const std::string &memorial_dir = FILENAMES["memorialdir"];
+    const std::string &memorial_dir = FILENAMES["memorialdir"] +  world_generator->active_world->world_name + "/";
```

Comment:
IIRC it used to vary as to precisely when it was accepted. For example FAT vs NTFS vs NFS. Its a long time since I had to use any of that so I'm not sure how much (if any) of that is still the case.

> There is no path_separator variable somewhere I assume?

Not sure on that one.

> So should work.

It should be easy to test. You just need to find someone to do it?